site-cookbooks/jenkins-java-trusty/recipes/default.rb

```diff
@@ -7,12 +7,7 @@
 # All rights reserved - Do Not Redistribute
 
 to_purge = %w(openjdk-6-jre-headless openjdk-6-jre-lib)
-to_install = %w(default-jre-headless openjdk-7-jre-headless)
-
-if Chef::VersionConstraint.new('= 16.04').include?(node['platform_version'])
-  to_purge = []
-  to_install = %w(default-jre-headless openjdk-8-jre-headless)
-end
+to_install = %w(default-jre-headless openjdk-8-jre-headless)
```

Comment:
According to a quick search bionic defaults to java11, so we should install the 11 jdk with a similar version constraint we had before.